routes/books.js

```diff
@@ -1,20 +1,23 @@
 var express = require("express");
 var router = express.Router();
 const db = require("../models");
+const isAuth = require("../middleware/authMiddleware").isAuth;
+const isNotAuth = require("../middleware/authMiddleware").isNotAuth;
 
-router.get("/add", async (req, res, next) => {
+router.get("/add", isAuth, async (req, res, next) => {
   try {
-    res.render("books/add");
+    res.render("books/add", { loggedin: req.isAuthenticated() ? true : false });
```

Comment:
You could just leave this with the `req.isAuthenticated()`, don't you?
Because if return true, you are returning true, otherwise false.

Reply:
Yeah but i send that variable to the template to render / or not something